docs/2deploy-microservices.adoc

```diff
@@ -75,7 +75,7 @@ kubectl create -f ../../kubernetes/Service.yml -n tutorial
 
 [source, bash]
 ----
-oc new-app --name=customer --context-dir=customer/java/springboot JAVA_OPTIONS='-Xms128m -Xmx256m -Djava.net.preferIPv4Stack=true' fabric8/s2i-java~https://github.com/redhat-developer-demos/istio-tutorial -o yaml  > customer.yml
+oc new-app --name=customer --context-dir=customer/java/springboot JAVA_OPTIONS='-Xms128m -Xmx256m -Djava.net.preferIPv4Stack=true' JAEGER_SERVICE_NAME=customer fabric8/s2i-java~https://github.com/redhat-developer-demos/istio-tutorial -o yaml  > customer.yml
```

Comment:
It seems that we need the -e flag for each variable.

Reply:
It seems that using `oc v3.9.27`, `-e` flag is not necessary. However we could add it for safer/cleaner command.